libkbfs/block_retrieval_queue.go

```diff
@@ -173,7 +173,7 @@ func (brq *blockRetrievalQueue) Request(ctx context.Context, priority int, kmd K
 			// specific type where the request blocks are CommonBlocks, but
 			// that direction can Set correctly. The cache will never have
 			// CommonBlocks.
-			cachedBlock, hasPrefetched, err :=
```

Comment:
Or maybe we need a `PutPermanentToTransient` which atomically removes an entry from the Permanent cache and adds it to the Transient cache.